source/dhtslib/sam.d

```diff
@@ -246,6 +408,143 @@ class SAMRecord
     @nogc @safe nothrow
     @property void insertSize(int isize) { this.b.core.isize = isize; }
 }
+immutable ubyte[256] seq_nt16_table = [
```

Comment:
this is already defined in hts.d (although not as immutable).
Why duplicate?

Reply:
Must have missed it when I looked through hts.d. Will address.